server/command/service.go

```diff
@@ -262,7 +276,8 @@ func (s *service) ExecuteCommand(pluginContext *plugin.Context, commandArgs *mod
 }
 
 func (s *service) handleMain(in *commandParams) (*model.CommandResponse, error) {
-	return s.runSubcommand(s.allCommands(), in)
+	conf := s.conf.GetConfig()
+	return s.runSubcommand(s.allCommands(conf), in)
```

Comment:
why is this called `allCommands` and not `allSubCommands`  (since we are calling `runSubcommand` on the set)?

Reply:
Good point, updated in 5a0d918376e50d705f1a0cdf349f90beddb0c2b3